README.md

```diff
@@ -51,6 +51,13 @@ Output:
 
 Learn More at [Syntax Guide](https://abelljs.org/syntax-guide)
 
+### 📄License
+This project is licensed under the MIT license. For more information on the license and its terms, please refer to the [LICENSE](./LICENSE) file.
+
+**Copyright (c) 2020 abelljs**
+
+If you have any questions or need further information about the license, please contact the project maintainers.
```

Comment:
Can we shorten this a bit to 1 line? Since not many people who come to see README would want to visit License link since LICENSE.md is a standard file for github repositories.

Maybe it can just be a small section that says-

> This project is licensed under the [MIT License](./License)